src/hooks/v2/V2ContractLoader.ts

```diff
@@ -46,11 +65,16 @@ export function useV2ContractLoader() {
   return contracts
 }
 
-const loadContract = (
-  contractName: keyof typeof V2ContractName,
+const loadContract = async (
```

Comment:
what's the benefit of async here? does `await import` enable the lazy loading?

Reply:
just improves webpack code splitting https://github.com/jbx-protocol/juice-interface/pull/1085#discussion_r884034706